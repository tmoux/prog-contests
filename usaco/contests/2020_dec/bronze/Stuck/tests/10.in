50
E 610506582 164826621
E 370917955 384370888
N 951426815 813274570
N 216220853 699460008
E 304811783 223712350
N 610009097 856363827
E 846621269 584522071
N 146533149 936060910
E 892430639 449228398
E 12836610 627992393
N 528433890 238498976
N 270754552 609416931
N 43388777 413360099
N 286448566 629580952
N 6072641 934392735
E 736491298 396918184
N 376696776 96055805
E 664423428 242588954
E 445080308 135019593
E 434322197 372555
E 916250774 528806445
E 415522325 799560997
N 446648412 695466127
E 143565421 981914693
E 539942439 987987334
N 12548159 576994985
N 489001354 953691761
E 402492972 470631541
E 104561852 915711850
E 108375482 202550399
N 887665154 118801173
N 610486506 386839851
E 942724790 833488263
N 688323172 829570037
N 916018859 222028828
N 78107280 234576987
E 342146590 723578341
E 849725352 978587665
N 599893069 83149517
N 190113083 44041351
E 928189300 931706506
E 900104667 394709364
N 671581032 337434154
E 991039875 878273679
N 292413412 794292538
N 434290636 724916170
E 153162844 67062760
N 825726814 769304465
E 968922326 221713886
N 880346039 411826969
