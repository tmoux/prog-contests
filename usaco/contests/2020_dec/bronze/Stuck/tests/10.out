277158572
157515935
20213693
216251842
295081286
131623507
Infinity
45853783
Infinity
257917942
232132565
306294919
214632294
286130898
Infinity
Infinity
127656545
223241726
154812761
Infinity
Infinity
410204489
104094870
345435933
285784375
Infinity
Infinity
207993534
187851560
268321294
278117011
412721146
Infinity
158417297
172680536
393415406
104501822
Infinity
387482024
158509048
Infinity
Infinity
462126843
Infinity
187622155
74644827
36950239
Infinity
Infinity
172695102
